example/provider-local/managedseeds/shoot-managedseed.yaml

```diff
@@ -12,13 +12,18 @@ spec:
   secretBindingName: local
   region: local
   networking:
-    type: calico
+    #type: calico
+    #providerConfig:
+    #  apiVersion: calico.networking.extensions.gardener.cloud/v1alpha1
+    #  kind: NetworkConfig
+    #  backend: none
+    #  typha:
+    #    enabled: false
```

Comment:
Remove this

Reply:
Addressed with https://github.com/gardener/gardener/pull/7664/commits/57da282be02f4408a9c52b0680cb95ed3913048d.